pom.xml

```diff
@@ -627,7 +632,7 @@
                         <version>${maven-replacer.version}</version>
                         <executions>
                             <execution>
-                                <id>replace-pre-keycloak22</id>
+                                <id>replace-pre-keycloak19</id>
```

Comment:
why 19?

Reply:
It is the replacer plugin within the Keycloak 19 profile. When we introduced that mechanism, we named the replacer plugins 22 and 23, someone copied but didn't change the name.